RubberduckTests/UnitTesting/PermissiveAssertTests.cs

```diff
@@ -0,0 +1,601 @@
+        }
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreSameShouldSucceedWithSameReferences()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    var obj1 = new object();
+        //    var obj2 = obj1;
+        //    assert.AreSame(obj1, obj2);
+
+        //    Assert.AreEqual(TestOutcome.Succeeded, _args.Outcome);
+        //}
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreSameShouldFailWithDifferentReferences()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    var obj1 = new object();
+        //    var obj2 = new object();
+        //    assert.AreSame(obj1, obj2);
+
+        //    Assert.AreEqual(TestOutcome.Failed, _args.Outcome);
+        //}
+
+        [Category("Unit Testing")]
+        [Test]
+        public void AreSameShouldSucceedWithTwoNullReferences()
+        {
+            var assert = new PermissiveAssertClass();
+            assert.AreSame(null, null);
+
+            Assert.AreEqual(TestOutcome.Succeeded, _args.Outcome);
+        }
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreSameShouldFailWithActualNullReference()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    assert.AreSame(new object(), null);
+
+        //    Assert.AreEqual(TestOutcome.Failed, _args.Outcome);
+        //}
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreSameShouldFailWithExpectedNullReference()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    assert.AreSame(null, new object());
+
+        //    Assert.AreEqual(TestOutcome.Failed, _args.Outcome);
+        //}
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreNotSameShouldSucceedWithDifferentReferences()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    var obj1 = new object();
+        //    var obj2 = new object();
+        //    assert.AreNotSame(obj1, obj2);
+
+        //    Assert.AreEqual(TestOutcome.Succeeded, _args.Outcome);
+        //}
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreNotSameShouldSuccedWithOneNullReference()
+        //{
+        //    var assert = new PermissiveAssertClass();
+        //    assert.AreNotSame(new object(), null);
+
+        //    Assert.AreEqual(TestOutcome.Succeeded, _args.Outcome);
+        //}
+
+        [Category("Unit Testing")]
+        [Test]
+        public void AreNotSameShouldFailWithBothNullReferences()
+        {
+            var assert = new PermissiveAssertClass();
+            assert.AreNotSame(null, null);
+
+            Assert.AreEqual(TestOutcome.Failed, _args.Outcome);
+        }
+
+        //[Category("Unit Testing")]
+        //[Test]
+        //public void AreNotSameShouldFailWithSameReferences()
```

Comment:
Why is there a commented out test? Delete it or, if it just does not work yet, ignore it using the corresponding attribute.